### DISCUSSION TITLE
Intel/AMD CPU - Submissions for EXPORT competition

### DISCUSSION THREAD

**Discussion by batrlatom** [ultralytics/yolov5#3432] [Show and tell]
## <div align="center">Compete and Win</div>

Use this discussion thread for YOLOv5 🚀 EXPORT Competition submissions in the **Intel/AMD CPU** category. Good luck!

<p align="center">
  <a href="https://github.com/ultralytics/yolov5/discussions/3213">
  <img width="850" src="https://github.com/ultralytics/yolov5/releases/download/v1.0/banner-export-competition.png"></a>
</p>

**Comment by SamSamhuns** [comment]
Are we allowed to submit an export solution using frameworks like Intel's openvino?

**Reply by batrlatom** [reply]
if the solution does not require the presence of a hardware accelerator such as an Intel stick, then it is possible

**Reply by nhcVigorous** [reply]
了

**Comment by SamSamhuns** [comment]
https://github.com/SamSamhuns/yolov5_export_cpu

Putting this repo for exporting the YOLOv5 model to OpenVINO's IR model format for Intel CPU/Edge device inference

**Reply by batrlatom** [reply]
Hi Sam, thank you for the submission. We will take a look at it

**Comment by karanjakhar** [comment]
https://github.com/karanjakhar/yolov5-export-to-cpu

Please have a look.

**Reply by batrlatom** [reply]
will do, thanks!

**Comment by AnhPC03** [comment]
https://github.com/AnhPC03/yolov5-export-to-cpu-mnn
My repository exports YOLOv5 model from Pytorch to MNN (a lightweight deep neural network inference engine) to run on Intel/AMD CPU easily.
Please take a look. Thank you.

**Reply by glenn-jocher** [reply]
@AnhPC03 very cool! Could you add profiling results for your MNN inference vs PyTorch inference? Preferably in a reproducible environment like Colab (even though it's not the fastest). Thanks!

**Reply by AnhPC03** [reply]
@glenn-jocher Thank you. I will add profiling results as soon as posible.

**Reply by AnhPC03** [reply]
@glenn-jocher Sorry, could I have one question? Did you mean **profiling results**? What's information do you want to know? Thanks.

**Reply by glenn-jocher** [reply]
@AnhPC03 ah profiling just means testing the inference speed

**Reply by AnhPC03** [reply]
@glenn-jocher I've compared inference speed between MNN and Pytorch in my [Google Colab](https://colab.research.google.com/drive/1V4wzynFtBEPsQk4bmmgbbSUZt-_6s69G?usp=sharing). I ran loop 50 times inference in one image and got following result: **MNN** took **52** seconds and **Pytorch** took **173** seconds. Please took a look the link above, and tell me if you want other information. Hope this is your need.

**Reply by batrlatom** [reply]
This looks like a quite speedup! I am looking forward for evaluation

**Comment by aditya-dl** [comment]
@shivamswarnkar and I have worked on this submission for AMD/Intel CPU. 

https://github.com/BlueMirrors/Yolov5-ONNX

We are getting identical outputs as PyTorch Inference(FP16). This uses letterboxing (so no need to forcefully resize) and you can directly test it on Colab as well. After benchmarks with 4 backends - torch, Tensorflow, tflite, and ONNX; ONNX performed the best for us. 

P.S. We are working on batching support.

**Comment by stefani-kovachevska** [comment]
Hi participants, 

Thank you all for your submissions, we have already reviewed them and are about to announce the winners! Please **send me an email** at [stefani@ultralytics.com](mailto:stefani@ultralytics.com) including your full name and GitHub username, so I can share more information with you. 

Stay tuned - the results are coming soon!

Stefani Kovachevska
_Project Manager at  Ultralytics_

**Comment by stefani-kovachevska** [comment]
Hi everyone, 

We were thrilled to have received your submissions to our YOLOv5 Export Competition for the Intel/AMD CPU category.

Unfortunately, the submissions in this category did not match the minimum set of requirements in each of the evaluation criteria. Therefore, no winner was selected for the category this time. However, there will be more chances for participants to compete again in the future. 

Thank you all for your efforts and for being a valuable member of our open-source community. Contributions from you are what makes our community great. 🙌 

Be sure to check out the winning repositories in the rest of the categories in our latest blog post: https://ultralytics.com/article/export-competition-winners

Stay amazing and keep creating! 🚀 
_Team Ultralytics_
